components/dropdown/dropdown.test.js

```diff
@@ -81,148 +62,132 @@ describe('DtDropdown Tests', () => {
     testContext.localVue = createLocalVue();
   });
 
-  // Teardown
-  afterEach(() => {
-    propsData = basePropsData;
-    slots = baseSlots;
-    scopedSlots = baseScopedSlots;
-    listeners = {};
-    wrapper.destroy();
-  });
-
   afterAll(() => {
     // Restore RequestAnimationFrame and cancelAnimationFrame
     global.requestAnimationFrame = undefined;
     global.cancelAnimationFrame = undefined;
   });
 
+  beforeEach(() => {
+    updateWrapper();
+  });
+
+  afterEach(() => {
+    mockProps = {};
+    mockSlots = {};
+    mockScopedSlots = {};
+    mockListeners = {};
+  });
+
   describe('Presentation Tests', () => {
-    // Test setup
-    beforeEach(() => {
-      _setWrappers();
+    it('should render the component', () => {
+      expect(wrapper.exists()).toBe(true);
     });
 
-    it(
-      'should render the component',
-      () => { expect(wrapper.exists()).toBe(true); },
-    );
+    it('should not render the visually hidden close button', () => {
+      const buttonExists = wrapper.findComponent(SrOnlyCloseButton).exists();
```

Comment:
Something like this would be better, or if you're reusing the component, add it to the `updateWrapper` function.

```suggestion
      const button = wrapper.findComponent(SrOnlyCloseButton);
      
      expect(button.exists()).toBe(false);
```

Reply:
Definetly, but Im just doing refactor test with the new test template.

Reply:
It's not that you're refactoring too much to fix that, I'd be cool if you can fix some stuff that you find even if it's not what you're currently doing.

That help us to have less tech-debt.

Reply:
Sure, tomorrow will retake this tests migrations and I will do a couple of improvements

Reply:
Updated!